src/plugins/intel_cpu/src/nodes/mkldnn_roi_align_node.h

```diff
@@ -13,12 +13,55 @@
 
 namespace MKLDNNPlugin {
 
+enum ROIAlignLayoutType {
+    ncsp,
+    blk,
+    nspc
+};
+
+struct jit_roi_align_params {
+    Algorithm alg;
+    InferenceEngine::Precision data_prc;
+    int data_size;
+    ROIAlignLayoutType layout;
+    int pooled_h;
+    int pooled_w;
+};
+
+struct jit_roi_align_call_args {
+    const void *src;
```

Comment:
const void **src?

Reply:
it's pointer to input data(const void *src) for planar layout and pointer to input data address list(const void **scr) for other layouts. I added code comments.